src/configure/helper/devOps/azureDevOpsHelper.ts

```diff
@@ -50,6 +59,11 @@ export class AzureDevOpsHelper {
             let parts = part.split('/');
             return parts[2].trim();
         }
+        else if (remoteUrl.indexOf(AzureDevOpsHelper.SSHAzureReposUrl) >= 0 || remoteUrl.indexOf(AzureDevOpsHelper.SSHVsoReposUrl) >= 0) {
```

Comment:
this logic seems repeated. You can have a separate parser class which returns an object containing org name, project name and repo name. And use same function in both places